src/apps/active-preview/Preview.js

```diff
@@ -191,23 +188,21 @@ export function Preview(props) {
               <Select
                 className={styles.Select}
                 name="device"
-                value="fullscreen"
-                onSelect={(val) => setDevice(val)}
+                value={device}
+                onChange={(e) => setDevice(e.target.value)}
```

Comment:
lets avoid single character variables.I would prefer this to be `evt` to express that it is an event